Makefile

```diff
@@ -1,48 +1,62 @@
 BINARY := go-spacemesh
 VERSION := 0.0.1
-COMMIT := $(shell git rev-parse HEAD)
-BRANCH := $(shell git rev-parse --abbrev-ref HEAD)
-BIN_DIR := $(shell pwd)/build
-CURR_DIR := $(shell pwd)
+COMMIT = $(shell git rev-parse HEAD)
+BRANCH = $(shell git rev-parse --abbrev-ref HEAD)
+BIN_DIR = $(shell pwd)/build
+CURR_DIR = $(shell pwd)
 export GO111MODULE = on
 
 # Setup the -ldflags option to pass vars defined here to app vars
-LDFLAGS := -ldflags "-X main.version=${VERSION} -X main.commit=${COMMIT} -X main.branch=${BRANCH}"
+LDFLAGS = -ldflags "-X main.version=${VERSION} -X main.commit=${COMMIT} -X main.branch=${BRANCH}"
 
-PKGS := $(shell go list ./...)
+PKGS = $(shell go list ./...)
 
-PLATFORMS := windows linux darwin
+platforms := windows linux darwin
```

Comment:
Either lower or upper just be consistent

Reply:
UPPER! 🙃 6123218